README.md

```diff
@@ -396,14 +396,71 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored in the `TokenManager`.
+
+By default:
+
+  - the user will be signed out using an API method 
+  - the page location will not be changed. 
+  
+When calling `signOut` with the default options, it is recommended to reload the page after the promise resolves to ensure all user data is cleared from memory and the app is properly loaded in an unauthenticated state.
+
+Since the API method relies upon an XHR call and the ability to set 3rd-party cookies, it may fail in certain scenarios, leaving the user's remote session open. For this reason, we generally recommend using the redirect method over the API method. The redirect method sets the window location to Okta where the session will be fully cleared.
```

Comment:
> The redirect method sets the window location to Okta where the session will be fully cleared.

How about "and works regardless of any browser settings for 3rd party cookies." ?

Reply:
Thats basically what I'm saying with "WILL BE FULLY CLEARED". I didn't want to dwell on the 3rd party cookie issue because there are other reasons an XHR could fail. The main point I want to make is that API MIGHT WORK whereas redirecting to Okta WILL WORK.

Reply:
I mention the 3rd party cookies because we've had devs that explicitly run into that bug, so this directs them to a solution and/or tells devs that it can become a bug, as opposed to a generally vague "might not work".

Reply:
Well thats why I mention 3rd party cookies as a reason for possible failure with the API method. We could add other possible failure cases, but in all these cases the answer would be to use the redirect method instead. So I think the goal is to steer everyone towards the redirect method, which (AFAIK) has no failure cases.